backoffice/views.py

```diff
@@ -47,6 +47,9 @@ def home(request):
                 RangeBoundary(),
             ),
         ).order_by("-reservation_time"),
+        "late_bookings": Booking.objects.filter(
```

Comment:
Unfortunately there's a little merge conflict here you'll need to resolve now, thanks to me merging some of your other PRs. You can do this by updating your `main` branch to the latest version from the ios-cv repo, and then merging that into your PR branch and resolving the conflicts. I can help you through it tomorrow if you like.

Reply:
Merged main and resolved conflicts.